iot_config/src/main.rs

```diff
@@ -71,14 +71,18 @@ impl Daemon {
             shutdown_trigger.trigger()
         });
 
+        let admin_pubkey = Arc::new(settings.admin_pubkey()?);
```

Comment:
Is an arc necessary here ?  Why not clone the key itself given it is a relatively small payload

Reply:
removed; changed the strategy here